### PR TITLE
create learning rate variable for every parameter

### LINKED ISSUE #5525: need to support setting different learning rate for each parameter

### PR DESCRIPTION
**Pull request by QiJune**
Fix #5525

### CODE REVIEW

File: python/paddle/v2/framework/optimizer.py [hunk 1]
```diff
@@ -35,15 +35,21 @@ def _append_optimize_op(self, block, param_and_grad):
...
+        param = param_and_grad[0]
+        param_lr = param.optimize_attr['learning_rate']
+        param_lr_shape = [1]
+        param_lr_var = self.helper.create_global_variable(
+            name=unique_name("learning_rate"),
+            dtype='float32',
+            shape=param_lr_shape,
+            lod_level=1,
+            persistable=True)
+        param_lr = param_lr * self._learning_rate
```
Comment (abhinavarora) [inline python/paddle/v2/framework/optimizer.py]: Are we sure that this will be multiplied? Is there a reference for this implementation or is it taken from the old Paddle code?

Comment (QiJune) [reply python/paddle/v2/framework/optimizer.py]: @lcy-seso @reyoung Could you please confirm this? Is each parameter's learning rate set relatively, and calculated by multiplying global learning rate?

Comment (lcy-seso) [reply python/paddle/v2/framework/optimizer.py]: I confirm this is the way old PaddlePaddle use. The learning rate of one parameter is determined by multiplying the global learning rate to the layerwise set learning rate (if set).

Comment (abhinavarora) [reply python/paddle/v2/framework/optimizer.py]: @lcy-seso Do you know how this works together with learning rate scheduler? Does the learning rate scheduler only work for the global learning rate?

Comment (lcy-seso) [reply python/paddle/v2/framework/optimizer.py]: From the codes, the learning rate schedule only works for global learning rate: 
- `calcLearningRate` adjusts the learning rate:
https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/parameter/FirstOrderOptimizer.h#L32 https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/parameter/ParameterOptimizer.h#L49
- Here takes Adam for example, how the final learning rate in `update` calculates:
https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/parameter/FirstOrderOptimizer.cpp#L272
